website/docs/r/alert_configuration.html.markdown

```diff
@@ -260,10 +260,14 @@ List of notifications to send when an alert condition is detected.
     - `USER`
     - `VICTOR_OPS`
     - `WEBHOOK`
+    - `MICROSOFT_TEAMS`
 
 * `username` - Name of the Atlas user to which to send notifications. Only a user in the project that owns the alert configuration is allowed here. Required for the `USER` notifications type.
 * `victor_ops_api_key` - VictorOps API key. Required for the `VICTOR_OPS` notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
 * `victor_ops_routing_key` - VictorOps routing key. Optional for the `VICTOR_OPS` notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
+* `webhook_secret` - Authentication secret for the `WEBHOOK` notifications type.
+* `webhook_url` - Target URL  for the `WEBHOOK` notifications type.
+* `microsoft_teams_webhook_url` - Microsoft Teams channel incoming webhook URL. Required for the `MICROSOFT_TEAMS` notifications type.
```

Comment:
Just adding a bit more of the info from https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createOneAlertConfigurationInOneProject.  Especially that we remove if invalid as that would cause a Terraform change.  

```suggestion
* `microsoft_teams_webhook_url` - Microsoft Teams Webhook Uniform Resource Locator (URL) that MongoDB Cloud needs to send this notification via Microsoft Teams. Required if `type_name` is `MICROSOFT_TEAMS`. If the URL later becomes invalid, MongoDB Cloud sends an email to the project owners. If the key remains invalid, MongoDB Cloud removes it. 
```

Reply:
Updated